pkg/resource/logging-credentials/logging_operator_secrets.go

```diff
@@ -21,6 +22,10 @@ const (
 	LoggingCredentialsNamespace = "monitoring"
 )
 
+type user struct {
```

Comment:
Should that struct be named credentials or userCredentials?

Reply:
Done.